spec/config/initializers/ab_tests_spec.rb

```diff
@@ -18,7 +10,7 @@ def reload_ab_test_initializer!
       allow(IdentityConfig.store).to receive(:idv_native_camera_a_b_testing_percent).
         and_return(percent)
 
-      reload_ab_test_initializer!
+      AbTests.reload_ab_test_initializer!
```

Comment:
[question] is there any advantage to using `described_class` here?

Reply:
Maybe? Won't hurt. Done in 88c5c646d